tests/data_context/datasource/test_data_context_datasource_runtime_data_connector_sparkdf_execution_engine.py

```diff
@@ -52,15 +52,12 @@ def data_context_with_datasource_spark_engine_batch_spec_passthrough(
 # Tests with data passed in as batch_data
 #########################################
 
-
 # Tests with SparkDFExecutionEngine : batch_data
-
-
-def test_get_batch_successful_specification_sparkdf_engine(
+def test_batch_data_get_batch_successful_specification_sparkdf_engine(
     data_context_with_datasource_spark_engine, spark_session, test_df_spark
 ):
     context = data_context_with_datasource_spark_engine
-    test_df = test_df_spark
+    test_df: "pyspark.sql.dataframe.DataFrame" = test_df_spark
```

Comment:
@Shinnnyshinshin not sure if we need quotes -- if we do, please append ` # noqa: F821`.  Thanks!

Reply:
We do here, because we aren't actually importing pyspark (it's loaded, along with a scope-check, at the fixture-level). Thank you for catching the `# noqa` addition. I've added it to the file